packages/react-native-renderer/src/ReactFabric.js

```diff
@@ -267,6 +268,8 @@ export {
   unmountComponentAtNode,
   stopSurface,
   createPortal,
+  // Used by internal Facebook production builds to support a UI review tool.
```

Comment:
```suggestion
  // Internal only.
```

Reply:
I hear what you're saying but this is a top-level (public) export so "internal only" isn't accurate. We could prefix it with one of those "__DO_NOT_USE" type strings if that's the intention.

Reply:
This comment is stripped out of the built bundle so it's probably nbd either way. Guess I prefer the more detailed comment to avoid confusion for us down the road, but I don't feel strongly.